library/src/main/java/org/xmtp/android/library/push/README.md

```diff
@@ -77,7 +81,24 @@ This example branch can serve as the basis for what you might want to provide fo
     ```
 
     ```kotlin
-    XMTPPush(context, "10.0.2.2:8080").subscribe(conversations.map { it.topic })
+    val subscriptions = conversations.map {
+        val hmacKeysResult = ClientManager.client.conversations.getHmacKeys()
+        val hmacKeys = hmacKeysResult.hmacKeysMap
```

Comment:
Want to move this out of subscriptions map?

Reply:
Good call 👍  Moved results out like the example has.